src/server/middleware/security/v2_token.go

```diff
@@ -67,7 +59,8 @@ func (vt *v2Token) Generate(req *http.Request) security.Context {
 		logger.Warningf("failed to decode bearer token: %v", err)
 		return nil
 	}
-	if err := t.Claims.Valid(); err != nil {
+	var v = jwt.NewValidator(jwt.WithLeeway(60 * time.Second), jwt.WithAudience(svc_token.Registry))
```

Comment:
This line failed with go fmt lint check
Please help to update it with below
```
var v = jwt.NewValidator(jwt.WithLeeway(60*time.Second), jwt.WithAudience(svc_token.Registry))
```

Reply:
fixed by e22a356, sorry for all these back and forth!